### PR TITLE
New template: WPComics

### PR DESCRIPTION
**Pull request by beerpiss**
Includes [NetTruyen (Vietnamese)](https://www.nettruyen.co) and [XOXO Comics (English)](https://xoxocomics.com).

### Issues
- Sometimes the chapters are numbered oddly (especially on XOXO Comics), so chapters might be displayed as a bunch of `Ch.0`s.
- XOXO Comics' genre filter is really limited; only one genre at any time can be chosen, so the workaround may look funny.

### CODE REVIEW

**Comment by beerpiss** [comment]
The title for these being blank was intentional, since both of these sources only displayed `Chapter <number>` and `<title> Issue #<number>`, which aren't exactly title-worthy imo

**Review by Skittyblock — CHANGES_REQUESTED** [review]
out directory should be gitignored'd and icons should be resized to 128x128

**Comment by beerpiss** [comment]
> out directory should be gitignored'd and icons should be resized to 128x128

done, I used imagemagick though so yell at me if it looks funny

**Review by Skittyblock — APPROVED** [review]

File: src/rust/wpcomics/template/src/template.rs [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+    let mut nsfw = MangaContentRating::Safe;
+    let mut viewer = MangaViewer::Default;
+    for node in details.select("li.kind.row > p.col-xs-8").text().read().split(" - ") {
+        let category = String::from(node);
+        if category == String::from("Smut") || category == String::from("Mature") || category == String::from("Adult") || category == String::from("18+") {
+            nsfw = MangaContentRating::Nsfw;
+        } else if category == String::from("Ecchi") || category == String::from("16+") {
+            nsfw = MangaContentRating::Suggestive;
+        }
+        if category.contains("Webtoon") {
```
Comment (Skittyblock) [inline src/rust/wpcomics/template/src/template.rs:71]: Manhwa and Manhua tags should use Scroll reader as well

```suggestion
        if category.contains("Webtoon") || category.contains("Manhwa") || category.contains("Manhua") {
```

File: src/rust/wpcomics/template/src/template.rs [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+pub fn get_manga_details(id: String, status_from_string: fn(String) -> MangaStatus, title_transformer: fn(String) -> String) -> Result<Manga> {
+    let details = Request::new(id.clone().as_str(), HttpMethod::Get).html();
+    let title = details.select("h1.title-detail").text().read();
+    let cover = append_protocol(details.select("div.col-image > img").attr("src").read());
+    let author = details.select("ul.list-info > li.author > p.col-xs-8").text().read();
+    let description = details.select("div.detail-content > p").text().read();
+    let mut categories = Vec::new();
+    let mut nsfw = MangaContentRating::Safe;
+    let mut viewer = MangaViewer::Default;
```
Comment (Skittyblock) [inline src/rust/wpcomics/template/src/template.rs:63]: Should use RTL by default.

```suggestion
    let mut viewer = MangaViewer::Rtl;
```

**Review by Skittyblock — CHANGES_REQUESTED** [review]

**Pull request merged**